sycl/test-e2e/lit.cfg.py

```diff
@@ -122,7 +122,7 @@
     for env_pair in config.extra_environment.split(","):
         [var, val] = env_pair.split("=", 1)
         if val:
-            llvm_config.with_environment(var, val, append_path=True)
+            llvm_config.with_environment(var, val, os.path.isabs(val))
```

Comment:
Looks like it was added at https://github.com/intel/llvm-test-suite/pull/531 and it doesn't provide much details. Are we 100% relative paths are never used here? Should we normalize them to absolute paths first?

Also, I'd keep the `append_path=` part to make clear what this boolean parameter means.

Reply:
> Should we normalize them to absolute paths first?

os.path.abspath can't be used here because any string will be parsed as a path relative to current working directory.

Reply:
First, important part was

> Are we 100% relative paths are never used here

Second, 

> os.path.abspath can't be used here because any string will be parsed as a path relative to current working directory.

I meant normalize after checking if the "string" points to an actual object on the filesystem.

And thinking about this more, `os.path.isabs` is a hacky heuristic to determine how the particular parameter should be handled. I don't like that at all. IMO, explicit list of what variables should be handled as `append_path` and what shouldn't would be way better than `isabs`.

Reply:
> First, important part was
> 
> > Are we 100% relative paths are never used here
> 

os.path.abspath is the api that converts relative path to abs path.

> Second,
> 
> > os.path.abspath can't be used here because any string will be parsed as a path relative to current working directory.
> 
> I meant normalize after checking if the "string" points to an actual object on the filesystem.

Why path must points to an actual object on the filesystem? Env could accept a path to an object that isn't created yet.

> And thinking about this more, `os.path.isabs` is a hacky heuristic

I'm not sure why os.path.isabs is hacky. There isn't really a good way to check if the input `val` is a path.

Reply:
Perhaps we should just delete append_path=* part from this line. It isn't safe to check if `val` is a path and it isn't safe to always append even if `val` is a path.
I suppose it is test driver's reponsibility to append when necessary before setting the env.
@aelovikov-intel what do you think?

Reply:
I'd be much happier with the removal, if you looked into what arguments come here and saw none that require `append_path` handling.

Reply:
Done removal. Update PR title and commit message accordingly. @aelovikov-intel please review again, thank you.